src/views/v2/list.php

```diff
@@ -53,6 +53,7 @@
 
 		<header <?php tribe_classes( $header_classes ); ?>>
 			<?php $this->template( 'components/messages' ); ?>
+			<?php $this->template( 'components/messages', [ 'classes' => [ 'tribe-events-header__messages--mobile' ] ] ); ?>
```

Comment:
G and I are looking at this in a call. We think you may have a file you haven't committed where `$classes` gets merged into the array of classes being output. `src/views/v2/components/messages.php`. We're guessing that the change just wasn't pushed up to this PR